File: Data Structures & Algorithms/max-consecutive-ones/submission-0.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int longest = 0;
        int current = 0;
        for(int i = 0; i < nums.length; i++){
            if (nums[i] == 0){
                current = 0;
            }
            else{
                current += 1;
            }
            if (longest < current){
                longest = current;
            }
            }
            return longest;
        }
    }

File: Data Structures & Algorithms/divide-array-into-equal-pairs/submission-0.py
class Solution:
    def divideArray(self, nums: List[int]) -> bool:
        count = {}
        for num in nums:
            count[num] = 1 + count.get(num, 0)
        
        for value in count:
            if count.get(value) % 2 != 0:
                return False
        
        return True